src/Events/Authorization/XacmlMapperHelper.cs

```diff
@@ -25,6 +25,8 @@ public static class XacmlMapperHelper
 
     // urn:altinn:organization:identifier-no is a value defined by Authorization so we need to use 'z' here.
```

Comment:
This comment on line 26 can be removed. We have concluded that 'z' is the way to go, no reason to explain why we use a 'z' notation moving forward.